contributors/devel/sig-testing/testing.md

```diff
@@ -46,13 +46,23 @@ passing, so it is often a good idea to make sure the e2e tests work as well.
 ### Run all unit tests
 
 `make test` is the entrypoint for running the unit tests that ensures that
-`GOPATH` is set up correctly.  If you have `GOPATH` set up correctly, you can
-also just use `go test` directly.
+`GOPATH` is set up correctly.
 
 ```sh
 cd kubernetes
 make test  # Run all unit tests.
 ```
+If you have `GOPATH` set up correctly, you can
```

Comment:
With a new enough `go`, do you need this?

Reply:
Doesn't this require the main repo to setup go modules?

Reply:
I agree that I think that unless go module support is added we still need the GOPATH to be setup.